Embrace Unity iOS Interface/EmbraceManager.swift

```diff
@@ -0,0 +1,494 @@
+                                  startInMillis: Double,
+                                  endInMillis: Double,
+                                  bytesSent: Double,
+                                  bytesReceived: Double,
+                                  statusCode: Double,
+                                  error: String?) {
+        var attributes = [
+            "http.request.method": httpMethod.uppercased(),
+            "url.full": url
+        ]
+        
+        if statusCode >= 0 {
+            attributes["http.response.status_code"] = String(Int(statusCode))
+        }
+
+        if bytesSent >= 0 {
+            attributes["http.request.body.size"] = String(Int(bytesSent))
+        }
+
+        if bytesReceived >= 0 {
+            attributes["http.response.body.size"] = String(Int(bytesReceived))
+        }
+        
+        if let error {
+            attributes["error.message"] = error
+        }
+        
+        Embrace.client?.recordCompletedSpan(name: createNetworkSpanName(url: url, httpMethod: httpMethod),
+                                            type: .networkRequest,
+                                            parent: nil,
+                                            startTime: convertDoubleToDate(ms: startInMillis),
+                                            endTime: convertDoubleToDate(ms: endInMillis),
+                                            attributes: attributes,
+                                            events: [],
+                                            errorCode: nil)
+    }
+    
+    static func logNetworkClientError(url: String,
+                                      httpMethod: String,
+                                      startInMillis: Double,
+                                      endInMillis: Double,
+                                      errorType: String,
+                                      errorMessage: String) {
+        // In case it matters, if we stylistically prefer to guard rather than letting optional chaning handling things we could do as below:
+        /*
+         guard let Embrace.client else {
+            // Possibly log the issue
+            return
+         }
+         */
+        Embrace.client?.recordCompletedSpan(name: createNetworkSpanName(url: url, httpMethod: httpMethod),
+                                            type: .networkRequest,
+                                            parent: nil,
+                                            startTime: convertDoubleToDate(ms: startInMillis),
+                                            endTime: convertDoubleToDate(ms: endInMillis),
+                                            attributes: [
+                                                "http.request.method": httpMethod.uppercased(),
+                                                "url.full": url,
+                                                "error.message": errorMessage,
+                                                "error.type": errorType
```

Comment:
Same as above. Here're the keys from `EmbraceSemantics`:
```suggestion
                                                SpanSemantics.NetworkRequest.keyMethod: httpMethod.uppercased(),
                                                SpanSemantics.NetworkRequest.keyUrl: url,
                                                SpanSemantics.NetworkRequest.keyErrorMessage: errorMessage,
                                                SpanSemantics.NetworkRequest.keyErrorType: errorType
```